main.nf

```diff
@@ -135,4 +135,3 @@ workflow {
      }
      
 }
-
```

Comment:
new line at end of file can be good in many situations

Reply:
sure, thanks!